docs/specification/draft/server/profiles.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: Profiles
+---
+
+<div id="enable-section-numbers" />
+
+<Info>**Protocol Revision**: draft</Info>
+
+The Model Context Protocol enables the definition of a collection of behaviors and properties as Profiles. Standard Profiles are available through the Model Context Protocol, and anyone may define custom Profiles. MCP Servers can declare the profiles they support. MCP Clients can discover which profiles an MCP Server supports, request the use of specific profiles, and negotiate a mutually agreed profile to use within a session with the MCP Server.
+
+## Profile Specification
+Each Profile has a specification document that describes the profile in detail. The profile specification is a document written using best practices prescribed in [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119) that unambiguously specifies the characteristics of a profile.
```

Comment:
Who defines the specification and where is it published? Do we expect that every single profile (you mentioned above that custom profiles can be defined) has a MCP spec associated with it?